packages/react-components/react-nav-preview/stories/NavDrawerFooter/index.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { NavDrawerFooter } from '@fluentui/react-nav-preview';
 import descriptionMd from './NavDrawerFooterDescription.md';
 import bestPracticesMd from './NavDrawerFooterBestPractices.md';
 
-export { Default } from './NavDrawerFooterDefault.stories';
+// export { Default } from './NavDrawerFooterDefault.stories';
```

Comment:
ditto remove and the ones below

Reply:
I'll want them back eventually when it's time to document them all as child stories to the flagship. This just hides them for the time being.

Reply:
I see the idea is to have them like Card has its subcomponents. I think you might be missing Hamburger and HamburgerInNav.

![image](https://github.com/microsoft/fluentui/assets/5953191/5355c3df-77b7-46ae-a6cc-8a26017cb444)


Another thing I noticed that should be fixed before releasing this package is to either add best practices and make them collapsible like the other components or remove the md file. Otherwise they appear blank:

![image](https://github.com/microsoft/fluentui/assets/5953191/50af6cc3-f01a-42a9-80a6-158fce8659e0)

Reply:
Great feedback, thanks Esteban, wilco!